src/Templates/src/templates/maui-mobile/Resources/Styles.xaml

```diff
@@ -374,7 +360,7 @@
 
     <Style TargetType="Page" ApplyToDerivedTypes="True">
         <Setter Property="Padding" Value="0"/>
-        <Setter Property="Background" Value="{AppThemeBinding Light={StaticResource WhiteBrush}, Dark={StaticResource BlackBrush}}" />
+        <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource WhiteBrush}, Dark={StaticResource BlackBrush}}" />
```

Comment:
BackgroundColor needs to be the color I think. Not sure there is a conversions from brush to color... But the brush may be a solid color so there may be.

Reply:
Ok, thanks! Changed it back

I hadn't been sure because I know both brushes and colors can be used for Background, and assumed it might be the same with Backgroundcolor